pdns/dnsdist-web.cc

```diff
@@ -443,6 +443,24 @@ static void connectionThread(int sock, ComboAddress remote)
           output << "\n";
         }
 
+        // Latency histogram buckets
+        output << "# HELP dnsdist_latency Histogram of responses by latency\n";
+        output << "# TYPE dnsdist_latency histogram\n";
+        output << "# HELP dnsdist_latency_bucket Number of queries contributing to response time histogram per latency bucket\n";
+        output << "# TYPE dnsdist_latency_bucket counter\n";
```

Comment:
I don't think we need "#HELP" and "#TYPE" lines for `dnsdist_latency_bucket`, based on https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md#text-format-example

Reply:
Removed the type and help lines for `dnsdist_latency_bucket`